src/structures/guild_preference.ts

```diff
@@ -932,21 +934,15 @@ export default class GuildPreference {
     async updateGuildPreferences(
         updatedOptionsObjects?: Array<{ name: string; value: GameOptionValue }>,
     ): Promise<void> {
-        interface GameOptionDatabaseEntry {
-            guild_id: string;
-            client_id: string;
-            option_name: string;
-            option_value: string;
-        }
-
-        let updatedOptions: GameOptionDatabaseEntry[] = [];
+        let updatedOptions: Insertable<GameOptionsSchema>[] = [];
         if (updatedOptionsObjects) {
             updatedOptions = Object.values(updatedOptionsObjects).map(
                 (option) => ({
                     guild_id: this.guildID,
                     client_id: process.env.BOT_CLIENT_ID as string,
                     option_name: option.name,
                     option_value: JSON.stringify(option.value),
+                    last_updated: new Date(),
```

Comment:
Nit: you might be able to default this to the current date on the table itself

Reply:
Its nullable